packages/did-core-test-server/suites/did-identifier/did-syntax.js

```diff
@@ -8,19 +8,6 @@ const didSyntaxTests = (suiteConfig) => {
           'ABNF Rules.', async () => {
           expect(didExample).toBeValidDid();
         });
-        it('The URI scheme MUST be "did:"', () => {
-          const url = new URL(didExample);
-          expect(url.protocol === 'did:').toBe(true);
-        });
-        it('The DID method name MUST be an ASCII lowercase string.', () => {
-          const method = didExample.split(':')[1];
-          expect(method).toBeAsciiString();
-          expect(method.toLowerCase()).toBe(method);
-        });
-        it('The DID method name MUST NOT be empty.', () => {
-          const method = didExample.split(':')[1];
-          expect(method).not.toBe('');
-        });
```

Comment:
The removal of these tests are correct -- they were replaced w/ new test language in the spec (and a test that covers all three of these cases).